src/main/java/com/google/api/codegen/discovery/transformer/nodejs/NodeJSSampleTypeNameConverter.java

```diff
@@ -26,6 +26,7 @@
   /** A map from primitive types in proto to NodeJS counterparts. */
   private static final ImmutableMap<Field.Kind, String> PRIMITIVE_TYPE_MAP =
```

Comment:
Currently PRIMITIVE_TYPE_MAP is used by getTypeNameForElementType, but IIUC this is only used by getTypeName, which I think uses it incorrectly, since Javascript arrays are untyped. I'm thinking neither PRIMITIVE_TYPE_MAP nor getTypeNameForElementType may be necessary.